### ISSUE TITLE
Handle lonely backslash in url.get tp avoid SyntaxError: JSON.parse: unterminated string error

### ISSUE THREAD

**Issue by AntonFriberg** [mattias-persson/url-parameters#1]
Hi!

There seems to be an issue with line https://github.com/mattias-persson/url-parameters/blob/master/index.js#L162 which leads to SyntaxError when a url contains parameters that includes a `\` character without any subsequent characters, e.g. `http://example.com/#?search=\`.

```javascript
// on visiting the http://example.com/#?search=\ url the following happens
// in Url.prototype.getParams function
var search = "search=%5C"
// this gets decoded in the following function
return JSON.parse('{"' + decodeURI(search).replace(/"/g, '\\"').replace(/&/g, '","').replace(/=/g,'":"') + '"}');
// however this is not valid since the result is this
return JSON.parse("{"search":"\"}")
// which is not valid JSON and results in SyntaxError: JSON.parse: unterminated string
```
This might just be an edge case which I happened to come across.

**Comment by mattias-sanfridsson** [comment]
Thanks for your report and sorry about the late response. I’ll take a look at this asap!

**Comment by mattias-sanfridsson** [comment]
Fixed in v1.6.1.
